keyboards/mechboards/lily58/pro/keyboard.json

```diff
@@ -0,0 +1,207 @@
+            {"matrix": [4, 4], "x": 87, "y": 64, "flags": 4},
+            {"matrix": [4, 3], "x": 65, "y": 64, "flags": 4},
+            {"matrix": [4, 2], "x": 51, "y": 62, "flags": 4},
+            {"matrix": [4, 1], "x": 36, "y": 62, "flags": 4},
+            {"x": 96, "y": 64, "flags": 2},
+            {"x": 32, "y": 64, "flags": 2},
+            {"x": 0, "y": 64, "flags": 2},
+            {"x": 0, "y": 32, "flags": 2},
+            {"x": 16, "y": 0, "flags": 2},
+            {"x": 50, "y": 0, "flags": 2},
+            {"x": 80, "y": 0, "flags": 2},
+            {"x": 96, "y": 32, "flags": 2},
+            {"matrix": [5, 5], "x": 152, "y": 4, "flags": 4},
+            {"matrix": [5, 4], "x": 166, "y": 2, "flags": 4},
+            {"matrix": [5, 3], "x": 181, "y": 0, "flags": 4},
+            {"matrix": [5, 2], "x": 195, "y": 2, "flags": 4},
+            {"matrix": [5, 1], "x": 210, "y": 6, "flags": 4},
+            {"matrix": [5, 0], "x": 224, "y": 8, "flags": 4},
+            {"matrix": [6, 0], "x": 224, "y": 23, "flags": 4},
+            {"matrix": [6, 1], "x": 210, "y": 21, "flags": 4},
+            {"matrix": [6, 2], "x": 195, "y": 17, "flags": 4},
+            {"matrix": [6, 3], "x": 181, "y": 15, "flags": 4},
+            {"matrix": [6, 4], "x": 166, "y": 17, "flags": 4},
+            {"matrix": [6, 5], "x": 152, "y": 19, "flags": 4},
+            {"matrix": [7, 5], "x": 152, "y": 34, "flags": 4},
+            {"matrix": [7, 4], "x": 166, "y": 32, "flags": 4},
+            {"matrix": [7, 3], "x": 181, "y": 30, "flags": 4},
+            {"matrix": [7, 2], "x": 195, "y": 32, "flags": 4},
+            {"matrix": [7, 1], "x": 210, "y": 36, "flags": 4},
+            {"matrix": [7, 0], "x": 224, "y": 38, "flags": 4},
+            {"matrix": [8, 0], "x": 224, "y": 53, "flags": 4},
+            {"matrix": [8, 1], "x": 210, "y": 51, "flags": 4},
+            {"matrix": [8, 2], "x": 195, "y": 47, "flags": 4},
+            {"matrix": [8, 3], "x": 181, "y": 45, "flags": 4},
+            {"matrix": [8, 4], "x": 166, "y": 47, "flags": 4},
+            {"matrix": [8, 5], "x": 152, "y": 49, "flags": 4},
+            {"matrix": [9, 5], "x": 137, "y": 41, "flags": 4},
+            {"matrix": [9, 4], "x": 137, "y": 64, "flags": 4},
+            {"matrix": [9, 3], "x": 159, "y": 64, "flags": 4},
+            {"matrix": [9, 2], "x": 173, "y": 62, "flags": 4},
+            {"matrix": [9, 1], "x": 188, "y": 62, "flags": 4},
+            {"x": 128, "y": 64, "flags": 2},
+            {"x": 192, "y": 64, "flags": 2},
+            {"x": 224, "y": 64, "flags": 2},
+            {"x": 224, "y": 32, "flags": 2},
+            {"x": 206, "y": 0, "flags": 2},
+            {"x": 150, "y": 0, "flags": 2},
+            {"x": 140, "y": 0, "flags": 2},
+            {"x": 128, "y": 32, "flags": 2}
+        ],
+        "max_brightness": 120,
+        "sleep": true,
+        "split_count": [37, 37]
+    },
+    "rgblight": {
+        "max_brightness": 120
+    },
```

Comment:
```suggestion
```

Reply:
This is a precaution in case someone decides to configure/enable rgblight.

I can remove it if required but it's intended to stop support questions.

Reply:
Probably fine to leave to leave, from a support standpoint. Although RGB Matrix keycodes being incompatible with those of RGBLight hopefully discourages using it.